### PR TITLE
Update references from BAMM to SAMM

### PR DESCRIPTION
**Pull request by bs-jokri**
Signed-off-by: Johannes Kristan <Johannes.kristan@bosch.com>

## Description
Updates references in the README from BAMM to SAMM and adds a reference to the currently used CLI version.

### CODE REVIEW

File: documentation/BAMM-method-documentation,-tool-and-training.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 
 Therefore, an Aspect model contains both information about runtime data structure (e.g., that there is a property in the data called "temperature", and that it has a numeric value) and information that is not part of the runtime data (e.g., the unit or range). It does not, however, contain actual runtime data (e.g., a numeric value representing the current temperature), as this will be delivered by an Aspect conforming to this Aspect model. The combination of raw runtime data and its corresponding Aspect model yields information.
-To specify an Aspect Model, the BAMM Aspect Meta Model (BAMM) [[bamm](https://openmanufacturingplatform.github.io/sds-bamm-aspect-meta-model/bamm-specification/snapshot/index.html)] is used.
+To specify an Aspect Model, the Semantic Aspect Meta Model (SAMM) [[samm](https://openmanufacturingplatform.github.io/sds-bamm-aspect-meta-model/bamm-specification/snapshot/index.html)] is used.
```
Comment (tunacicek) [inline documentation/BAMM-method-documentation,-tool-and-training.md:20]: Link to https://openmanufacturingplatform.github.io/sds-bamm-aspect-meta-model/bamm-specification/snapshot/index.html
is not working.

Comment (bs-jokri) [reply documentation/BAMM-method-documentation,-tool-and-training.md:20]: fixed. thx

**Comment by tunacicek** [comment]
LGTM

**Comment by tunacicek** [comment]
@bs-jokri  fine for me.

**Pull request merged**